docs/esa/iso.rst

```diff
@@ -350,7 +350,7 @@ Images can be displayed by using the following code:
   >>> from matplotlib.colors import LogNorm
   >>> 
   >>> #We configure the plot to be interactive
-  >>> %matplotlib widget
+  >>> # matplotlib widget
```

Comment:
I don't think these are strictly necessary changes (or sufficient to make tests pass). As a follow-up/enhancement PR it would be superb if you could take care of switching on the doctesting on the narrative docs. We made the switch for a couple of modules already, and some are in the works, but esa.iso/esa.hubble/esa.jwst were not amongst them.

Reply:
I was wrong, here is a PR that maybe suitable to use as a starter for the docs testing enabling, though it needs a rebase and significant cleanup work to have fewer lines ignored, etc.: https://github.com/astropy/astroquery/pull/1970